thoth/storages/graph/dgraph.py

```diff
@@ -2032,3 +2034,53 @@ def sync_dependency_monkey_result(self, document: dict) -> None:
             Resolved.from_properties(source=dependency_monkey_run, target=inspection_software_stack).get_or_create(
                 self.client
             )
+
+    def get_number_of_each_vertex_in_graph(self) -> dict:
+        """Retrieve dictionary with number of vertices per vertex label in the graph database."""
+        node_labels = [model.get_label() for model in ALL_MODELS if issubclass(model, VertexBase)]
+        tot_nodes_per_label = {}
+        for node_label in node_labels:
+            query = (
+                """
+            {
+                f(func: has(%s)) {
+                    c:count(uid)
+                }
+            }
+            """
+                % node_label
+            )
+            result = self._query_raw(query)
```

Comment:
This can be rewritten to have one request with multiple queries to dgraph or perform multiple requests in parallel (see `_query_raw_parallel`). Let's merge this and see if we will have some perf drawbacks by doing it this way.